### ISSUE TITLE
数据集

### ISSUE THREAD

**Issue by 153zml** [Complex-data/MUSER#1]
你好，感谢你在该领域做的工作，我是一名在校研究生，对您的研究内容很感兴趣，请问可以私发一下您的数据集吗？任何您的帮助我将不胜感激

**Comment by init-neok** [comment]
同求

**Comment by hedarko20** [comment]
同求

**Comment by Complex-data** [comment]
Regarding datasets, we offer the following suggestions:
1. The PolitiFact and GossipCop datasets were sourced using the FakeNewsNet tool, as described in the research by 
Kai Shu, Deepak Mahudeswaran, Suhang Wang, Dongwon Lee, and Huan Liu in their 2020 paper titled "Fakenewsnet: A Data Repository with News Content, Social Context, and Spatiotemporal Information for Studying Fake News on Social Media," published in Big Data, Volume 8, Issue 3, pages 171–188.
2. To validate the efficacy of our method, we have also utilized the LIAR dataset, which is accessible here https://www.cs.ucsb.edu/~william/data/liar_dataset.zip. Additionally, we employed the Fakeddit dataset, available on https://github.com/entitize/Fakeddit.
